### ISSUE TITLE
Picture "details "page reappear when it shouldn't.

### ISSUE THREAD

**Issue by OlivierS1** [nextcloud/android#1221]
### Actual behaviour
-Tell us what happens
Once the "details" from a picture has been requested, the page will always reappear when switching application back and forth.

### Expected behaviour
-Tell us what should happen
"Details" page should not reappear once it has been closed.

### Steps to reproduce
1. Go down a folder with pictures.
2. Click on the picture to display it.
3. Click the 3 vertical dots on the top right menu and select "Details"
4. Pressing the back arrow to return to the picture in picture view mode. (it actually show for a split second the folder then switch back to picture view mode).
5. Change folder by pressing back arrow.
6. Switch to another app on the device not closing NC Dev.
7. Switch back to NC Dev.
7. First thing NC Dev will show is the previous picture detail page. Presseing arrow back will point to the previous folder as well.

### Environment data
Android version:

Device model: 
LG G3 D852
Stock or customized system:
Stock
Nextcloud app version:
NC Dev 20170723
Nextcloud server version:
NC 12

#### Web server error log
```
Insert your webserver log here
```

#### Nextcloud log (data/nextcloud.log)
```
Insert your Nextcloud log here
```
**NOTE:** Be super sure to remove sensitive data like passwords, note that everybody can look here! You can use the Issue Template application to prefill some of the required information: https://apps.nextcloud.com/apps/issuetemplate

**Comment by tobiasKaminsky** [comment]
Confirmed, it happens also on master, so not dev related.

**Comment by jsrck** [comment]
I tried to fix the bug. Could you please verify it on #1231?

**Comment by AndyScherzinger** [comment]
Merged to master for 1.5.0
